<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChangePasswordDialog</class>
 <widget class="QDialog" name="ChangePasswordDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ChangePassword</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>25</x>
     <y>25</y>
     <width>250</width>
     <height>350</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="Modify">
    <property name="geometry">
     <rect>
      <x>25</x>
      <y>230</y>
      <width>90</width>
      <height>28</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>宋体</family>
     </font>
    </property>
    <property name="text">
     <string>修改</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Cancel">
    <property name="geometry">
     <rect>
      <x>135</x>
      <y>230</y>
      <width>90</width>
      <height>28</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>宋体</family>
     </font>
    </property>
    <property name="text">
     <string>取消</string>
    </property>
   </widget>
   <widget class="QLabel" name="Title">
    <property name="geometry">
     <rect>
      <x>65</x>
      <y>45</y>
      <width>120</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>等线</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>XXX数据库</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="NewPasswordAgain">
    <property name="geometry">
     <rect>
      <x>45</x>
      <y>180</y>
      <width>160</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>宋体</family>
     </font>
    </property>
    <property name="text">
     <string>确认新密码</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="NewPassword">
    <property name="geometry">
     <rect>
      <x>45</x>
      <y>140</y>
      <width>160</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>宋体</family>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>新密码</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="OldPassword">
    <property name="geometry">
     <rect>
      <x>45</x>
      <y>100</y>
      <width>160</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>宋体</family>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>旧密码</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
